gui/confirm.lua

```diff
@@ -31,47 +28,30 @@ function Opts:init()
             cursor_pen = COLOR_WHITE,
             choices = {},
             on_submit = self:callback('toggle'),
-            on_submit2 = self:callback('toggle_all'),
+        },
+        widgets.HotkeyLabel{
+            frame = {b=1, l=0},
+            label='Toggle all',
+            key='CUSTOM_ALT_E',
+            on_activate=function() self:toggle_all(self.subviews.list:setSelected(i)) self:refresh() end,
         },
         widgets.HotkeyLabel{
             frame = {b=2, l=0},
             label='Resume paused confirmations',
             key='CUSTOM_P',
-            on_activate=function() confirm.unpause() self:refresh() end,
-            enabled=function() return self.paused end},
+            on_activate=function() self:unpause_all() self:refresh() end,
+            enabled=function() return self.any_paused end
+        },
         widgets.Label{
             view_id = 'controls',
             frame = {b = 0, l = 0},
             text = {
-                {key = 'SELECT', text = ': Toggle, '},
-                {key = 'LEAVESCREEN', text = ': Back', on_activate = self:callback('dismiss')},
-                NEWLINE,
-                {key = 'SEC_SELECT', text = ': Toggle all'},
+                {key = 'SELECT', text = ': Toggle'},
             },
```

Comment:
To allow this hotkey hint to be clickable too, it should be a HotkeyLabel that runs `toggle` on SELECT